src/Altinn.Platform/Altinn.Platform.Register/Register/Controllers/PartiesController.cs

```diff
@@ -46,48 +46,6 @@ public async Task<ActionResult<Party>> Get(int partyID)
             return Ok(result);
         }
 
-        /// <summary>
-        /// Gets the party for a given social security number or organization number.
-        /// </summary>
-        /// <param name="lookupValue">SSN or OrgNumber.</param>
-        /// <returns>The party represeting the provided SNN/OrgNumber.</returns>
-        [HttpGet("lookupObject")]
-        [Consumes("application/json")]
-        [ProducesResponseType(404)]
-        [ProducesResponseType(200)]
-        [Produces("application/json")]
-        public async Task<ActionResult<Party>> LookupPartyBySSNOrOrgNo([FromBody]string lookupValue)
```

Comment:
This function shouldn't be deleted unless app backend is changed to use PostPartyLookup.

Reply:
App template updated in PR #4118. Resolving comment.